src/frontend/components/SøknadsSteg/Oppsummering/OppsummeringSteg/OmBarnet/OmBarnetOppsummering.tsx

```diff
@@ -330,27 +328,24 @@ const OmBarnetOppsummering: React.FC<Props> = ({ settFeilAnchors, nummer, barn,
                                 barn[barnDataKeySpørsmål.søkerForTidsromStartdato].svar
                             )}
                         />
-                        <OppsummeringFelt
-                            tittel={
-                                <SpråkTekst
-                                    id={
-                                        omBarnetSpørsmålSpråkId[
-                                            OmBarnetSpørsmålsId.søkerForTidsromSluttdato
-                                        ]
-                                    }
-                                />
-                            }
-                            søknadsvar={formaterDato(
-                                barn[barnDataKeySpørsmål.søkerForTidsromSluttdato].svar
-                            )}
-                        />
+                        {barn[barnDataKeySpørsmål.søkerForTidsromSluttdato].svar !==
+                            AlternativtSvarForInput.UKJENT && (
```

Comment:
Da kan man også her bare sjekke om spørsmålet har et svar, for hvis det ikke har et svar (tom string) så har det ikke blitt vist til brukeren